c++/src/capnp/schema.h

```diff
@@ -341,6 +454,12 @@ class StructSchema::Field {
   //
   // If the above does not make sense, you probably don't need this method.
 
+  schema::Value::Reader getDefaultValue() const {
```

Comment:
Can we name this `getDefaultValueProto()`, since it returns the capnp encoding of the default value and not a `DynamicValue`?

Reply:
Updated